2000-08-09  Jakub Jelinek  <jakub@redhat.com>

	* time/tzfile.c (__tzfile_compute): __tzstring zones
	from zone_names.

--- glibc-2.1.3/time/tzfile.c.jj	Fri May  7 16:41:44 1999
+++ glibc-2.1.3/time/tzfile.c	Fri Aug 25 09:55:20 2000
@@ -1,4 +1,4 @@
-/* Copyright (C) 1991,92,93,95,96,97,98,99 Free Software Foundation, Inc.
+/* Copyright (C) 1991,92,93,95,96,97,98,99,2000 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -498,7 +498,7 @@ __tzfile_compute (time_t timer, int use_
 	/* There is no daylight saving time.  */
 	__tzname[1] = __tzname[0];
       tp->tm_isdst = info->isdst;
-      tp->tm_zone = &zone_names[info->idx];
+      tp->tm_zone = __tzstring (&zone_names[info->idx]);
       tp->tm_gmtoff = info->offset;
     }
 
